crates/voicevox_core/src/user_dict/dict.rs

```diff
@@ -0,0 +1,85 @@
+use derive_getters::Getters;
+use std::{collections::HashMap, fs::File};
+use uuid::Uuid;
+
+use super::word::*;
+use crate::{Error, Result};
+
+/// ユーザー辞書。
+#[derive(Clone, Debug, Default, Getters)]
+pub struct UserDict {
+    words: HashMap<Uuid, UserDictWord>,
+}
+
+impl UserDict {
+    /// ユーザー辞書を作成する。
+    pub fn new() -> Self {
+        Default::default()
+    }
+
+    /// ユーザー辞書をファイルから読み込む。
+    /// ファイルが存在しない、または内容が不正の場合はエラーを返す。
```

Comment:
`//`ではなく`///`で補足するなら、Rustdocの慣習に合わせた方がよいかと。

```suggestion
    /// ユーザー辞書をファイルから読み込む。
    ///
    /// ファイルが存在しない、または内容が不正の場合はエラーを返す。
```